src/components/v5/common/Filter/partials/FilterOptions.module.css

```diff
@@ -0,0 +1,7 @@
+.li {
+  @apply flex items-center gap-3 transition-all duration-normal text-md mr-2 hover:bg-gray-50 hover:font-medium cursor-pointer rounded p-2;
+}
+
+.mobileRow {
+  @apply text-xs font-medium text-gray-400 uppercase;
```

Comment:
`text-4` is the same like `text-xs font-medium`
Please use directly to the className because you can not custom tailwind classes via module.css